sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,137 @@
+      title: "build Service Catalog binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        unset GOPATH; cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/hack/build-go.sh
+    - type: "script"
+      title: "run Service Catalog unit tests"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog test-unit
+    - type: "script"
+      title: "run Service Catalog integration tests"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        sudo yum -y install etcd
+        # this strips the -i arg to "go test". can be dropped once PR is merged:
+        # https://github.com/kubernetes-incubator/service-catalog/pull/1566 (and vendored in rebase)
+        sed -i 's/\(go test.*\)\( -i \)\(.*\)/\1 \3/' cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/test/integration.sh
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog scBuildImageTarget=.scBuildImage test-integration
+    - type: "script"
+      title: "build Service Catalog image"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        hack/build-local-images.py service-catalog
+    - type: "script"
+      title: "build Service Catalog E2E test binary"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        NO_DOCKER=1 make -C cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog  bin/e2e.test
+    - type: "script"
+      title: "start cluster with Service Catalog"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc cluster up --version=latest --service-catalog
+    - type: "script"
+      title: "Run Service Catalog E2E"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        sudo chmod a+rw /var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        export KUBECONFIG=/var/lib/origin/openshift.local.config/master/admin.kubeconfig
+        ./_output/local/bin/linux/amd64/oc login -u system:admin
+        SERVICECATALOGCONFIG=$KUBECONFIG cmd/service-catalog/go/src/github.com/kubernetes-incubator/service-catalog/bin/e2e.test  -v 10 -alsologtostderr 
+################################################################################
+# ASB CI ADDITIONS
+################################################################################
+    - type: "script"
+      title: "Install ASB to cluster"
+      repository: "origin"
+      timeout: 1800
+      script: |-
+        ./_output/local/bin/linux/amd64/oc create project ansible-service-broker
+        curl "https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/simple-broker-template.yaml" | ./_output/local/bin/linux/amd64/oc process -n "ansible-service-broker" -f - | ./_output/local/bin/linux/amd64/oc create -f -
```

Comment:
(This is where my expertise ends.)
You may consider adding a retry to curl download attempts.